spec/hook_spec.rb

```diff
@@ -89,4 +89,38 @@
       end
     end
   end
+
+  stores = ['Shark', 'Siga', 'Vale Mais', 'Vision', 'Guitierrez', 'Qi', 'Sjc', 'Abs', 'Capital', 'Geração de Cadastro']
+
+  stores.each do |store|
```

Comment:
What do you think about this? Is good for testing several cases without extending the code too much.

Reply:
This is not a good practice . You're testing the logic that splits the source get the value after "|" and insert into a product therefore it doesn't matter what is the value after the "|"